airmozilla/popcorn/templates/popcorn/editor.html

```diff
@@ -23,6 +23,7 @@
 {% endblock %}
 
 {% block full_width_content %}
+<slug data-url="{{ slug }}"></slug>
 <div id="editor"></div>
```

Comment:
Yes and no. It's a matter of foresight. Like with a global `styles.css` you *could* add a selector there that is only needed on one page. But there's a big chance you decommission that page in the far future but forget to delete it from `styles.css` and then you have a bloated `styles.css` that future developers get uneasy about cleaning up. That's why I favor using `<style>` tags on individual pages that only need a specific set of rules. (If it gets too big I wrote a thing like `eventedit.css` instead of an inline tag in the HTML)

Then, if I find a need a thing on more than just one page. I can either reuse `eventedit.css` on multiple pages. Or if it gets too many pages, it gets promoted to be included in the global `styles.css`. 

Does that help to explain the issue?

Basically, what if we change how cornpop.js works and nothing no longer needs that `data-slug="..."` in `main_base.html` then `main_base.html` gets scarily bloated.